tvos/CountlyTestApp-tvOS/AppDelegate.m

```diff
@@ -16,7 +16,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     config.host = @"https://YOUR_COUNTLY_SERVER";
```

Comment:
Hooray, I've caught you! You missed making the change in that line. :)

Reply:
Oops, how is it that possible :)